pkg/endpoint/endpoint_test.go

```diff
@@ -153,6 +154,9 @@ func TestBadCNValiateRequest(t *testing.T) {
 	if err == nil {
 		t.Fatalf("CN should not have matched")
 	}
+	if !strings.HasSuffix(err.Error(), "common name vcert.test.bonjo.com is not allowed in this policy: [.*.vfidev.com .*.venafi.com]") {
```

Comment:
This is an unrelated test improvement that makes sure that we get the expected error (before fixing getBaseZoneConfiguration, the returned error was actually the wrong error).